arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts

```diff
@@ -819,6 +819,7 @@ spi10_cs_pins: &spi10_cs_gpio1 {};
 		random = <&random>, "status";
 		rtc = <&rpi_rtc>, "status";
 		rtc_bbat_vchg = <&rpi_rtc>, "trickle-charge-microvolt:0";
+		sd_cqe = <&sdio1>, "supports-cqe";
```

Comment:
If the aim is to create a boolean (empty property when true, absent when empty) then `"supports-cqe?"` is the correct syntax.

Reply:
Yes, the property only has to exist for it to have an effect